Grinnell-DB-iOS/ResultsViewController.m

```diff
@@ -19,282 +19,281 @@ @implementation ResultsViewController
-        ProfileViewController *destViewController = segue.destinationViewController;
-        Person *selected = [[Person alloc] init];
-        selected = [searchDetails objectAtIndex:indexPath.row];
-        NSUInteger index = [selected.attributes indexOfObject:@"profileURL"];
-        if (NSNotFound != index) {
-            [self parseProfilePage:[selected.attributeVals objectAtIndex:index] forPerson:selected];
-        }
-        destViewController.selectedPerson = selected;
-    }
+  if ([segue.identifier isEqualToString:@"PushToProfile"]) {
+    NSIndexPath *indexPath = [self.tableView indexPathForSelectedRow];
+    [self.tableView deselectRowAtIndexPath:indexPath animated:YES];
+    ProfileViewController *destViewController = segue.destinationViewController;
+    destViewController.selectedPerson = [self personForIndexPath: indexPath];
+  }
 }
 
 //Method to determine the availability of network Connections using the Reachability Class
 - (BOOL)networkCheck {
-    Reachability *networkReachability = [Reachability reachabilityForInternetConnection];
-    NetworkStatus networkStatus = [networkReachability currentReachabilityStatus];
-    return (!(networkStatus == NotReachable));
+  Reachability *networkReachability = [Reachability reachabilityForInternetConnection];
+  NetworkStatus networkStatus = [networkReachability currentReachabilityStatus];
+  return (!(networkStatus == NotReachable));
 }
 
 #pragma mark - Table view data source
 - (NSInteger)numberOfSectionsInTableView:(UITableView *)tableView {
-    return 1;
+  return 1;
 }
 
 - (NSInteger)tableView:(UITableView *)tableView numberOfRowsInSection:(NSInteger)section {
-    return searchDetails.count;
+  return searchDetails.count;
 }
 
 - (CGFloat)tableView:(UITableView *)tableView heightForRowAtIndexPath:(NSIndexPath *)indexPath {
-    return 88;
+  return 88;
 }
 
 - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(NSIndexPath *)indexPath {
-    // Register the NIB cell object for our custom cell
-    if (onCampusBool) {
-        [tableView registerNib:[UINib nibWithNibName:@"OnCResultsCell" bundle:nil] forCellReuseIdentifier:cellIdentifier];
-    } else {
-        [tableView registerNib:[UINib nibWithNibName:@"OffCResultsCell" bundle:nil] forCellReuseIdentifier:cellIdentifier];
+
+  // Register the NIB cell object for our custom cell
+  if (onCampusBool) {
+    [tableView registerNib:[UINib nibWithNibName:@"OnCResultsCell" bundle:nil] forCellReuseIdentifier:cellIdentifier];
+  } else {
+    [tableView registerNib:[UINib nibWithNibName:@"OffCResultsCell" bundle:nil] forCellReuseIdentifier:cellIdentifier];
+  }
+
+  UITableViewCell *cell = (UITableViewCell*)[tableView dequeueReusableCellWithIdentifier:cellIdentifier];
+
+  if (!cell) {
+    cell = [[UITableViewCell alloc] initWithStyle:UITableViewCellStyleDefault reuseIdentifier:cellIdentifier];
+  }
+
+  // Connect the cell's properties
```

Comment:
Agreed. I'm still working on the rewrite, so much of this diff is actually just whitespace transformation. Can I fix this in a future branch?